G/gperftools/build_tarballs.jl

```diff
@@ -24,28 +24,26 @@ elif [[ "${target}" == *-apple-* ]]; then
     # https://github.com/Homebrew/homebrew-core/blob/5e8ab5f092bd4dfe9658bab6d86150e26a326de3/Formula/gperftools.rb#L28
     export CXXFLAGS=-D_XOPEN_SOURCE
 elif [[ "${target}" == *-freebsd* ]]; then
-    # Fix the error
-    #   undefined reference to `backtrace_symbols'
+    # Fix the error: undefined reference to `backtrace_symbols'
     export LDFLAGS="-lexecinfo"
+    export CPPFLAGS="-I${includedir}"
 fi
 
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+./configure \
+--prefix=${prefix} \
+--build=${MACHTYPE} \
+--host=${target} \
+--enable-libunwind=yes \
+--enable-static=no \
+--enable-shared=yes
+
 make -j${nproc}
 make install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [
-    Platform("i686", "linux"; libc="glibc"),
-    Platform("x86_64", "linux"; libc="glibc"),
-    Platform("aarch64", "linux"; libc="glibc"),
-    Platform("armv7l", "linux"; libc="glibc"),
-    Platform("powerpc64le", "linux"; libc="glibc"),
-    Platform("x86_64", "macos"),
-    Platform("x86_64", "freebsd")
-]
-platforms = expand_cxxstring_abis(platforms)
+platforms = expand_cxxstring_abis(supported_platforms(; exclude=Sys.iswindows))
 
```

Comment:
I don't think we can do much for aarch64-linux-musl, judging by the error messages in the log I think it's an upstream bug.  We never built for that platform before, I'd just filter it out
```suggestion
filter!(p -> !(Sys.islinux(p) && arch(p) = 64 && libc(p) == "musl"), platforms)
```

Reply:
That sounds good, I thought that might be the case. The same thing happened to me on windows, I took one or two looks at it and then just filtered it out